scripts/release_scripts/deploy.py

```diff
@@ -299,15 +314,25 @@ def flush_memcache(app_name):
     common.open_new_tab_in_browser_if_possible(memcache_url)
     common.ask_user_to_confirm('Please flush the memcache.')
 
-    admin_misc_tab_url = None
-    if app_name == APP_NAME_OPPIASERVER:
-        admin_misc_tab_url = 'https://www.oppia.org/admin#/misc'
-    else:
-        admin_misc_tab_url = 'https://%s.appspot.com/admin#/misc' % app_name
+    admin_misc_tab_url = get_admin_misc_tab_url(app_name)
+    common.open_new_tab_in_browser_if_possible(admin_misc_tab_url)
+    common.ask_user_to_confirm('Please flush the cache on Oppia website.')
 
-    if admin_misc_tab_url:
-        common.open_new_tab_in_browser_if_possible(admin_misc_tab_url)
-        common.ask_user_to_confirm('Please flush the cache on Oppia website.')
+
+def verify_number_of_pending_deletion_requests(app_name):
+    """Verify that the number of PendingDeletionRequestModels is zero.
+
+    Args:
+        app_name: str. The name of the app to deploy.
+    """
+    common.open_new_tab_in_browser_if_possible(get_admin_misc_tab_url(app_name))
+    common.ask_user_to_confirm(
+        'Please check that the number of PendingDeletionRequestModels '
+        'models is zero (you need to navigate to the '
+        '\'Number of pending deletion requests\' section). '
+        'If it is not zero the /cron/users/user_deletion and '
+        '/cron/users/fully_complete_user_deletion jobs will need '
```

Comment:
Are there instructions on how to do this manually? Might need to add to the release process or add more instructions in this prompt.

Reply:
These changes were moved to a private repo, let's continue the discussion there.